autorest/jsonrpc/__init__.py

```diff
@@ -111,7 +111,7 @@ def list_inputs(self) -> List[str]:
         """
 
     @abstractmethod
-    def get_value(self, key: str) -> str:
+    def get_value(self, key: str) -> Optional[str]:
```

Comment:
It's actually Optional[Any], since we know at least this can be bool, or empty dict